components/UserImage.vue

```diff
@@ -1,6 +1,12 @@
 <template>
   <v-avatar v-if="userImage">
-    <v-img :src="userImage" :alt="$auth.user.Name" class="userImage absolute" />
+    <v-img :src="userImage" :alt="$auth.user.Name" class="userImage absolute">
```

Comment:
```suggestion
    <v-img :src="userImage" :alt="$auth.user.Name" class="userImage">
```

This is some testing bits I forgot to remove in my PR, so if you can ammend your commit and do a force push it would be great :)

Reply:
Done :)